looprpc/client.proto

```diff
@@ -212,6 +212,29 @@ message LoopOutRequest {
     triggering the swap (loop CLI, autolooper, LiT UI, other 3rd party UI).
     */
     string initiator = 14;
+
+    /*
+    An alternative destination address source for the swap. This field
+    represents the name of the account in the backing lnd instance.
+    Refer to lnd's wallet import functions for reference.
+     */
+    string account = 15;
+
+    /*
+    The address type of the account specified in the account field.
+     */
+    AddressType account_addr_type = 16;
```

Comment:
Q: So given that LND's `ListAccounts` RPC also returns the address type, I'm wondering if we at all need this? Would also simplify a great bit on the following commits.

Reply:
Are you asking about reusing the type `AddressType` from lnd? Or are you asking if the user can omit this parameter entirely?
If the latter then no, we have to specify it since there can exists two accounts in lnd with equal names but different address types.

Reply:
Ah ok, thanks for clarifying, indeed I was asking for the latter and yes it makes sense now.